docs/index.bs

```diff
@@ -1336,28 +1339,35 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
     <section algorithm="wait-until-method">
       <h4 id="wait-until-method">{{ExtendableEvent/waitUntil()|event.waitUntil(f)}}</h4>
 
-      {{ExtendableEvent/waitUntil()}} method extends the lifetime of the event.
+      Note: {{ExtendableEvent/waitUntil()}} method extends the lifetime of the event.
 
       <dfn method for="ExtendableEvent"><code>waitUntil(|f|)</code></dfn> method *must* run these steps:
 
-        1. If the {{Event/isTrusted}} attribute is false, [=throw=] an "{{InvalidStateError}}" {{DOMException}}.
+        1. Let |event| be the [=context object=].
+        1. [=ExtendableEvent/Add lifetime promise=] |f| to |event|.
```

Comment:
I moved this into its own algorithm as we have it copy-pasted in the spec twice, and it's something other specs may want to do.

Reply:
Looks nicer.